src/App.js

```diff
@@ -1,25 +1,45 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from 'react';
+import { useDispatch} from 'react-redux';
+import { fetchWeather } from './actions';
+import Charts from './components/Charts';
 
 function App() {
```

Comment:
It may be slightly redundant to have `App.js` and `index.js` with the way you have things set up, but that's a bit of a nitpick on my part.